scripts/publish-prerelease

```diff
@@ -17,13 +17,13 @@ if !(command -v gh > /dev/null); then
   exit 1
 fi
 
-# hostname is necessary just in case you are logged into the CLI
-# GitHub enterprise instance
-if !(gh auth status --hostname "github.com" > /dev/null 2>&1); then
-  echo "Not logged into github".
-  echo "Please run: gh auth login"
-  exit 1
-fi
+## hostname is necessary just in case you are logged into the CLI
+## GitHub enterprise instance
+#if !(gh auth status --hostname "github.com" > /dev/null 2>&1); then
+#  echo "Not logged into github".
+#  echo "Please run: gh auth login"
+#  exit 1
+#fi
```

Comment:
All commented-out code is related to GitHub releases and will be added once our service account is approved. It will be manually handled until then.